services/order.py

```diff
@@ -0,0 +1,34 @@
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, User
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None) -> None:
+
+    with transaction.atomic():
+        user = User.objects.get(username=username)
+        order = Order.objects.create(user=user)
+
+        if date:
+            order.created_at = date
+
+        for ticket_data in tickets:
+            Ticket.objects.create(
+                order_id=order.id,
+                movie_session_id=ticket_data["movie_session"],
+                row=ticket_data["row"],
+                seat=ticket_data["seat"])
+
+        order.save()
+
+
+def get_orders(username: str = None) -> QuerySet:
+    if username:
+        user = User.objects.get(username=username)
+        return Order.objects.filter(user=user)

```

Comment:
You can filter by related name. No need to get a user. Read how to filter by related fields